data/json/mapgen/shelter.json

```diff
@@ -60,6 +61,47 @@
   {
     "type": "mapgen",
     "method": "json",
+    "weight": 100,
+    "om_terrain": [ "shelter_under" ],
+    "object": {
+      "rows": [
+        "########################",
+        "########################",
+        "########################",
+        "########################",
+        "####;;;;;;;;;;;;;;;;####",
+        "####;bbbbbbbbbbbbbb;####",
+        "####;b____________b;####",
+        "####;b____________b;####",
+        "####;b____________b;####",
+        "####;b___|-++-|___b;####",
+        "####;b___|____|___b;####",
+        "####;b___+_<<_+___b;####",
+        "####;b___+_<<_+___b;####",
+        "####;b___|____|___b;####",
+        "####;b___|-++-|___b;####",
+        "####;b____________b;####",
+        "####;b____________b;####",
+        "####;b____________b;####",
+        "####;bbbbbbbbbbbbbb;####",
+        "####;;;;;;;;;;;;;;;;####",
+        "########################",
+        "########################",
+        "########################",
+        "########################"
+      ],
+      "palettes": [
+        "shelter"
+      ],
+      "items": {
+        "_": { "item": "shelter", "chance": 5 }
```

Comment:
Been playing on this for a bit – think `"chance": 1` or `2` would more closely match the previous spawning. Really wasn't very much.